packages/website/modules/docs-theme/index.scss

```diff
@@ -10,6 +10,10 @@
   background-size: cover;
   word-break: break-word;
 
+  #navigation {
```

Comment:
This CSS targets the navigation element setting the word-break to normal / no break.